crates/vm-superio/src/serial.rs

```diff
@@ -542,6 +546,13 @@ impl<T: Trigger, EV: SerialEvents, W: Write> Serial<T, EV, W> {
             }
             // We want to enable only the interrupts that are available for 16550A (and below).
             IER_OFFSET => self.interrupt_enable = value & IER_UART_VALID_BITS,
+            FCR_OFFSET => {
+                if value & FCR_FIFO_RESET_RX != 0 || value & FCR_FIFO_RESET_TX != 0 {
```

Comment:
I'm a bit confused by this.  Resetting the RX buffer isn't the same thing as resetting the TX buffer.